app/upgrades/v17/upgrades_test.go

```diff
@@ -121,8 +123,10 @@ func (s *UpgradeTestSuite) checkCommunityPoolICAAccountsRegistered(chainId strin
 	expectedDepositPortId, _ := icatypes.NewControllerPortID(depositOwner)
 	expectedReturnPortId, _ := icatypes.NewControllerPortID(returnOwner)
 
-	depositPortIdRegistered := s.App.ICAControllerKeeper.IsBound(s.Ctx, expectedDepositPortId)
-	returnPortIdRegistered := s.App.ICAControllerKeeper.IsBound(s.Ctx, expectedReturnPortId)
+	scopedICAControllerKeeper := s.App.CapabilityKeeper.ScopeToModule(icacontrollertypes.SubModuleName)
```

Comment:
can you use `s.App.ScopedICAControllerKeeper`?

Reply:
nice 2dcced3b9260d2b0197cc41c6626efd264b536f0